index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sive's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+    <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <a href="#" id="logo" class="name-logo">Sive</a>
+        <nav class="navbar">
+            <img src="icons/union.png" alt="humburger" class="humburger">
+            <ul id="nav-items">
+                <li class="nav-item"><a href="#">Portfolio</a></li>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- Header Section -->
+    <section class="header-info-secion">
+        <h1 class="greeting-hearder">I’m Sive <br> Glad to see you!</h1>
+        <p class="greeting-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="connect"><h4>LET'S CONNECT</h4></a>
+
+        <ul id="social-icons">
+            <li class="github">
+                <img src="icons/Git.png" alt="GitHub Icon">
+            </li>
+            <li class="linkedin">
+                <img src="icons/Linkedin.png" alt="LinkedIn Icon">
+            </li>
+            <li class="angel">
+                <img src="icons/AngelList.png" alt="AngelList Icon">
+            </li>
+            <li class="twitter">
+                <img src="icons/Twitter.png" alt="Twitter Icon">
+            </li>
+            <li class="medium">
+                <img src="icons/Medium.png" alt="Medium Icon">
+            </li>
+        </ul>
+    </section>
+
+    <!-- Work Section -->
+    <section class="work">
+        <div class="card-container">
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/227616198-21a60dca-3160-454e-8026-28587ea0f8c9.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/227617595-c31b5fb5-b978-49a8-a183-35fc3c51953b.png)

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), we should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Please kindly adjust the spacing (margin/padding) taking the `background color` of the section into consideration so that it covers the required areas to match the template.